os/mm/Kconfig

```diff
@@ -154,3 +154,11 @@ config MM_ASSERT_ON_FAIL
 	---help---
 		When memory allocation fails, make assertion.
 		From the application binary with loadable build, it will be reloaded.
+
+config MM_DUMP_CORRPUTED_HEAP
+	bool "Dump the complete corrupted heap"
+	default n
+	---help---
+		When we find a heap corruption, we can dump the entire heap, so that
```

Comment:
I think moving "so that" to next line gives better readability.

Reply:
Thanks. But, I have adjusted the line length according to the kconfig file. It might be displayed differently in the menuconfig based on the screen size. So, let us keep the current format.